### PR TITLE
Adding a11y testing addon & partial fixes

### PR DESCRIPTION
**Pull request by sivakumar-kailasam**
Addresses #166. There are two pending issue categories. One for the select control not having a label & another on color contrast. I thought of replacing x-select with power-select for consistency with the guides. But wanted to get a confirmation here.

Our color combination for links [fails contrast checks](http://webaim.org/resources/contrastchecker/?bcolor=fffdf9&fcolor=dd6a58). The ember-a11y team has been trying to organize efforts on this via https://github.com/ember-a11y/issues/issues/1. So we might need to decide on those with their help. 

//cc @acorncom - since you were already involved on this conversation 🙂

### CODE REVIEW

File: app/styles/base/_variables.scss [hunk 1]
```diff
@@ -29,7 +29,7 @@ $dark-gray: #444545;
 $tan: #fffdf9;
 $white: #fff;
 $black: #000;
-$creme: #FFFBF5;
+$creme: #FDFDFD;
```
Comment (toddjordan) [inline app/styles/base/_variables.scss]: Not a color guy, but doesn't this change the color of the background of the entire app?   Maybe we should have a ux designer look at our palette and suggest something with proper contrast.  In fact I think people have looked at this for the website.  

@locks Has the color palette been part of the website design updates that have happened lately?

Comment (sivakumar-kailasam) [inline app/styles/base/_variables.scss]: Oh sorry for not mentioning this, but this particular color was taken off the guides 😄

Comment (toddjordan) [inline app/styles/base/_variables.scss]: oh, well, that'll work then.  I'm pulling the updates and running locally.  We'll see how it looks :-)

**Comment by toddjordan** [comment]
This is cool, but maybe we should add the config section to environment.js and have audit off by default.  I think the screen display is going to troll new contributors (it suprised me :-P )

And then make a followup issue that would have someone turn it on, clean up all existing issues, and then leave it enabled.

**Comment by sivakumar-kailasam** [comment]
Sure, adding that.

**Comment by toddjordan** [comment]
I resolved conflicts and merged with master, but the pr didn't like what I did for some reason.  Therefore I'm going to close the PR, since the code has been merged.

**Pull request closed without merging**